ApnsPHP/Push.php

```diff
@@ -433,12 +434,10 @@ protected function httpInit(): bool
     {
         $this->logger->info("Trying to initialize HTTP/2 backend...");
 
+        # curl_init() without arguments only really returns false in OOM scenarios,
+        # or an error in the DNS resolver initialization. phpstan says we can
+        # skip the false check, and I tend to agree.
         $this->hSocket = curl_init();
-        if ($this->hSocket === false) {
-            throw new Exception(
-                "Unable to initialize HTTP/2 backend."
-            );
-        }
```

Comment:
This is the only change I'm still a bit hesitant about :-/
The OOM situations are fine, we can ignore those. Not sure about the DNS resolver setup though

Reply:
I think with a DNS resolver issue APNS would not be on the top of the priority list.